lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -6,15 +6,16 @@
             'gettext',
             'js/views/fields',
             'teams/js/models/team',
+            'common/js/components/utils/view_utils',
             'text!teams/templates/edit-team.underscore'],
-        function (Backbone, _, gettext, FieldViews, TeamModel, editTeamTemplate) {
+        function (Backbone, _, gettext, FieldViews, TeamModel, ViewUtils, editTeamTemplate) {
             return Backbone.View.extend({
 
                 maxTeamNameLength: 255,
                 maxTeamDescriptionLength: 300,
 
                 events: {
-                    'click .action-primary': 'createOrUpdateTeam',
+                    'click .action-primary': function(event) {this.disableRun(event, this.createOrUpdateTeam);},
```

Comment:
The value of operation is a promise, not the function. Your disableRun function is evaluating the function and getting the promise and passing that to ``disableElementWhileRunning``. As it is defined, it accepts a function which when evaluated will return a promise. 

Coding with promises is always like this...